custom/icds_reports/utils/__init__.py

```diff
@@ -899,21 +899,45 @@ def create_aww_performance_excel_file(excel_data, data_type, month, state, distr
     worksheet.title = "AWW Performance Report"
     worksheet.sheet_view.showGridLines = False
     # sheet title
-    worksheet.merge_cells('B2:{0}2'.format(
-        "J" if aggregation_level == 3 else ("K" if aggregation_level == 2 else "L")
-    ))
+    if beta:
+        worksheet.merge_cells('B2:{0}2'.format(
+            "K" if aggregation_level == 3 else ("L" if aggregation_level == 2 else "M")
+        ))
+    else:
+        worksheet.merge_cells('B2:{0}2'.format(
+            "J" if aggregation_level == 3 else ("K" if aggregation_level == 2 else "L")
+        ))
     title_cell = worksheet['B2']
     title_cell.fill = PatternFill("solid", fgColor="4472C4")
     title_cell.value = "AWW Performance Report for the month of {}".format(month)
     title_cell.font = Font(size=18, color="FFFFFF")
     title_cell.alignment = Alignment(horizontal="center")
 
     # sheet header
-    header_cells = {"B3", "C3", "D3", "E3", "F3", "G3", "H3", "I3", "J3"}
-    if aggregation_level < 3:
-        header_cells.add("K3")
-    if aggregation_level < 2:
-        header_cells.add("L3")
+    if beta:
+        header_cells = {"B3", "C3", "D3", "E3", "F3", "G3", "H3", "I3", "J3", "K3"}
+        if aggregation_level < 3:
+            header_cells.add("L3")
+        if aggregation_level < 2:
+            header_cells.add("M3")
+        date_description_cell_start = "I3" if aggregation_level == 3 else \
+            ("J3" if aggregation_level == 2 else "K3")
+        date_description_cell_finish = "J3" if aggregation_level == 3 else \
+            ("K3" if aggregation_level == 2 else "L3")
+        date_column = "K3" if aggregation_level == 3 else ("L3" if aggregation_level == 2 else "M3")
+
+    else:
+        header_cells = {"B3", "C3", "D3", "E3", "F3", "G3", "H3", "I3", "J3"}
```

Comment:
Do you know why these are sets not lists. It looks like we only use them by iterating over them once, so a list should be fine and will be simpler (plus ordering is ensured)

Reply:
No Idea why they were added. Perhaps because here these all are like keys and order does not matter. But anyway gonna change it to List. Set looks unnecessarily used.